vimiv/api/settings.py

```diff
@@ -315,8 +316,51 @@ def __str__(self) -> str:
         return "String"
 
 
-# Initialize all settings
+class OrderSetting(Setting):
+    """Stores an ordering setting."""
+
+    typ = str
+
+    ORDER_TYPES: Dict[str, Tuple[Callable[..., Any], bool]] = {
+        "alphabetical": (str, False),
+        "alphabetical-inverse": (str, True),
+        "natural": (natural_sort, False),
+        "natural-inverse": (natural_sort, True),
+        "case-insensitive": (lambda s: s.lower(), False),
```

Comment:
`str.lower` is sufficient, there is no need for a lambda function.

Reply:
Nice! Will apply. Never seen it used that way though :smile: i.e. always `text.lower()` not `str.lower(text)`